acceptance/bundle/integration_whl/python-params/databricks.yml.tmpl

```diff
@@ -2,12 +2,12 @@ bundle:
   name: wheel-task-with-environments
```

Comment:
Is this the same as acceptance/bundle/integration_whl/serverless but with parameters?

Any way we can share the configs between the two and apply BundleConfig to apply the differences?

At the very least we should follow the same name convention: e.g. serverless, serverless_with_parameters